uktrade/data-hub-frontend
# Pipeline list

## Description of change

This PR branches off of https://github.com/uktrade/data-hub-frontend/pull/2677 to continue to build out the `my-pipeline` app. This PR introduces the `PipelineItem` and `PipelineList` components, places them in the `my-pipeline` app and adds the requisite apis call, redux state management and functional testing. 

## Test instructions

Check out the branch and point to the sandbox api. Click on the `My pipeline` tab on the home page and you will see some mock pipeline items. The edit button won't lead anywhere yet as that is coming in a later PR. 

## Screenshots

![image](https://user-images.githubusercontent.com/42253716/82334848-700f5b00-99e0-11ea-888d-e8d3d764f7ed.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
